backend/src/resolvers/user_management.js

```diff
@@ -100,6 +100,26 @@ export default {
 
         return encode(currentUser)
       }
+    },
+    addSocialMedia: async (_, { url }, { driver, user }) => {
```

Comment:
@mattwr18 I would consider to create an extra node for `SocialMedia` which contains little but a `url`. This would have a couple of advantages:
* You can utilize `neo4j-graphql-js` for parts of the resolver
* You can add unique constrains on `SocialMedia.url`
* You can use [contraint-directives](https://github.com/confuser/graphql-constraint-directive)
* You can have counts on social media links

Reply:
> @mattwr18 I would consider to create an extra node for `SocialMedia` which contains little but a `url`. This would have a couple of advantages:
> 
> * You can utilize `neo4j-graphql-js` for parts of the resolver
> * You can add unique constrains on `SocialMedia.url`
> * You can use [contraint-directives](https://github.com/confuser/graphql-constraint-directive)
> * You can have counts on social media links

I need to ask for some clarification on this @roschaefer... if I understand what you mean correctly, you are suggesting
1. create a `type` called `SocialMedia`, "which contains little but a url"
2. create an relation between `User` and `SocialMedia`, when a user sends a link to add their social media?
3. Add the constraints listed above, i.e. unique urls, limited amount of social media links

Is this what you are suggested, or did I misunderstand?

Reply:
Yes! Correct. Don't worry too much about the constraints though. Maybe two people share a link? (e.g. an organization or sth.) same same with the number of SocialMedia.

It's just that I believe you get most out of Neo4J if you model relationships through relationships not through properties.

Reply:
Ah, by `Counts on SocialMedia` I meant a computed property like `socialMediaCount` and it was just an example for sth. you can do with `neo4j-graphql-js` easily which you can't do with properties so easily.